pkgs/by-name/im/immersed-vr/package.nix

```
@@ -1,27 +1,44 @@
 { lib
 , appimageTools
+, callPackage
 , fetchurl
+, stdenv
 }:
-appimageTools.wrapType2 rec {
+let
   pname = "immersed-vr";
   version = "9.10";
   name = "${pname}-${version}";
 
-  src = fetchurl {
-    url = "https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
-    hash = "sha256-Mx8UnV4fZSebj9ah650ZqsL/EIJpM6jl8tYmXJZiJpA=";
+  sources = {
+    x86_64-linux = {
+      url = "https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
+      hash = "sha256-vYk7WHGVOzrrep6vmA58ELa6aDsZFUw5D2StCYP0Ioc=";
+    };
+    aarch64-linux = {
+      url = "https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
+      hash = "sha256-HuhoGG2aQ2NcbZlQfUmGUl0IvhXPO5uV7x4WKJRR7Ew=";
+    };
```

```suggestion
    x86_64-linux = {
      url = "https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage";
      hash = "sha256-Mx8UnV4fZSebj9ah650ZqsL/EIJpM6jl8tYmXJZiJpA=";
    };
```

The hash still seems to be `sha256-Mx8UnV4fZSebj9ah650ZqsL/EIJpM6jl8tYmXJZiJpA=` here.

---

How does one discover these without building?
Tried:
```
> curl https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage | shasum -a 256 -b | base64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  102M  100  102M    0     0  1894k      0  0:00:55  0:00:55 --:--:--  825k
MzMxZjE0OWQ1ZTFmNjUyNzliOGZkNmExZWI5ZDE5YWFjMmZmMTA4MjY5MzNhOGU1ZjJkNjI2NWM5
NjYyMjY5MCAqLQo=
❯ nix-prefetch-url https://web.archive.org/web/20240210075929/https://static.immersed.com/dl/Immersed-x86_64.AppImage
path is '/nix/store/xs5pav2cagari4rvm2zwznxy09w3h7ii-Immersed-x86_64.AppImage'
1416cab5q9nnybjshcv9h88gzhma36fyp8fniydjfr8zbsfi87rk
```

---

https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-store-prefetch-file